draft-westerlund-tsvwg-dtls-over-sctp-bis.md

```diff
@@ -508,40 +508,93 @@ ULP:  Upper Layer Protocol
    KeyUpdate. See {{sec-Consideration}} for security considerations
    regarding rekeying.
 
+### DTLS 1.2 Considerations
+
+   Before sending during renegotiation a ClientHello message or ServerHello
+   message, the DTLS endpoint MUST ensure that all DTLS messages using the
+   previous epoch have been acknowledged by the SCTP peer in a non-revokable
+   way.
```

Comment:
For me a DTLS message consists of a DTLS record. In case of UDP. In case of UDP, you can put multiple DTLS records or DTLS messages in a single UDP packet. In case of SCTP you can put multiple DTLS messages in a single SCTP user message. This is what this document is about. RFC 6083 has a one-to-one relation between DTLS messages / DTLS records and SCTP user messages.

Are you asking for a wording like `MUST ensure that all DATA chunks related to DTLS messages using the previous epoch have been`?